### DISCUSSION TITLE
Modal template slots?

### DISCUSSION THREAD

**Discussion by edwh** [cdmoro/bootstrap-vue-3#387] [Q&A] [answered]
I have a `b-modal` which uses `<template slot="default" slot-scope="{ cancel }">`.  When this is rendered, the DOM contains a `template` node and the modal is apparently empty.  

![image](https://user-images.githubusercontent.com/56696/168134501-b46af5c7-8140-477c-8f48-ecbc2c4d9463.png)

If I move the contents of the template node up, then stuff starts to show.

I wouldn't expect the `template` to make it through into the DOM.  But it's possible that I'm doing something wrong here, and that the way this is supposed to be done has changed in Vue 3.  `b-modal` isn't in the docs yet, so that doesn't help.  Any advice?

**Comment by edwh** [comment, marked as answer]
The answer is that you need to use this kind of syntax:

`        <template #default>
`

I think `slot` was deprecated in favour of `v-slot` a while back, and `v-slot` is now deprecated in favour of `#`.  But I could be wrong.

**Reply by Hiws** [reply]
> `v-slot` is now deprecated in favour of `#`

This isn't the case.
The `#`, is the shorthand for `v-slot`.
Same as `@` being the shorthand for `v-on` and `:` being the shorthand for `v-bind`
